bftengine/src/communication/TlsTCPCommunication.cpp

```diff
@@ -666,40 +648,45 @@ class AsyncTlsConnection : public enable_shared_from_this<AsyncTlsConnection> {
   }
 
   void connect_timer_tick(const B_ERROR_CODE &ec) {
-    LOG_TRACE("enter, node " << _selfId << ", dest: " << _destId << ", ec: "
-                             << ec.message());
+    LOG_TRACE(_logger,"enter, node " << _selfId
+        << ", dest: " << _destId
+        << ", ec: " << ec.message());
 
     if (_closed) {
-      LOG_DEBUG(
-          "closed, node " << _selfId << ", dest: " << _destId << ", ec: "
-                          << ec.message());
+      LOG_DEBUG(_logger, "closed, node " << _selfId
+         << ", dest: " << _destId
+         << ", ec: " << ec.message());
     } else {
       if (connected) {
```

Comment:
Don't `connected` and `_connectedTimer` need to be protected by a lock like they are elsewhere?